jena-fuseki2/jena-fuseki-webapp/pom.xml

```diff
@@ -56,6 +56,12 @@
       <artifactId>jena-text</artifactId>
       <version>3.17.0-SNAPSHOT</version>
     </dependency>
+    
+    <dependency>
+        <groupId>org.apache.jena</groupId>
+        <artifactId>jena-tdb2</artifactId>
+        <version>3.17.0-SNAPSHOT</version>
+    </dependency>
```

Comment:
This addition was necessary to be able to call `DatabaseMgr.compact()`

Reply:
core has a dependency on jena-tdb2 via apache-jena-libs.

I also check by adding a call to `DatabaseMgr.compact` in a class jena-fuseki-webapp.

`(cd jena-fuseki-webapp ; mvn dependency:tree)` on the default branch of the codebase gives me:
```
[INFO] org.apache.jena:jena-fuseki-webapp:jar:3.17.0-SNAPSHOT
[INFO] +- org.apache.jena:jena-fuseki-core:jar:3.17.0-SNAPSHOT:compile
[INFO] |  +- org.apache.jena:apache-jena-libs:pom:3.17.0-SNAPSHOT:compile
[INFO] |  |  +- org.apache.jena:jena-shacl:jar:3.17.0-SNAPSHOT:compile
[INFO] |  |  |  \- org.apache.jena:jena-arq:jar:3.17.0-SNAPSHOT:compile
[INFO] |  |  +- org.apache.jena:jena-tdb:jar:3.17.0-SNAPSHOT:compile
[INFO] |  |  +- org.apache.jena:jena-tdb2:jar:3.17.0-SNAPSHOT:compile
```

Reply:
Yeah I can probably take this out, I was having some issues with code completion in Eclipse and thought I was missing a dependency